src/main/java/marquez/service/JobService.java

```diff
@@ -44,6 +45,8 @@
   private final JobRunDao jobRunDao;
   private final JobRunArgsDao jobRunArgsDao;
 
+  private static final Job.Type DEFAULT_JOB_TYPE = Job.Type.BATCH;
```

Comment:
Would we want to default to a job type to `BATCH`?

Reply:
We should make the job type required when registered a job.

Reply:
This was the recommendation of @sshah-wework (specifically that it be optional and default to BATCH). 
(from slack):
> we can start w/ it being optional, and it defaults to BATCH

Reply:
Filed #567 to adjust the behavior. Likely a Q3 task.